engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeErrorsTest.scala

```diff
@@ -1133,6 +1176,212 @@ class RuntimeErrorsTest
     context.consumeOut shouldEqual List()
   }
 
+  it should "send updates when panic is resolved in method" in {
```

Comment:
I would suggest rephrasing this (and the next one's) tests' names - the word 'resolve' is unclear to me in this context.

Maybe something like 'when the node does not result in a panic after an update' or something like that?